acceptance/suites/tests/code_commands/code_scripts.rb

```diff
@@ -137,43 +161,28 @@ class { '::ntp': }
   on master, "cd #{git_local_repo} && git commit -m 'commit to setup r10k example'"
   on master, "cd #{git_local_repo} && git push origin production"  
   on master, "/opt/puppetlabs/puppet/bin/r10k deploy environment -p"
+end
 
-  
-step 'SETUP: Install the code-id-command script'
+step 'SETUP: Install the code-id-command script' do
   code_id_command_script=<<-CIC
   #!/usr/bin/env sh  
   /opt/puppetlabs/puppet/bin/r10k deploy display -p --detail $1 | grep signature | grep -oE '[0-9a-f]{40}'
   CIC
   create_remote_file(master, '/opt/puppetlabs/server/apps/puppetserver/code-id-command_script.sh', code_id_command_script)
   on(master, 'chown puppet:root /opt/puppetlabs/server/apps/puppetserver/code-id-command_script.sh')
   on(master, 'chmod 770 /opt/puppetlabs/server/apps/puppetserver/code-id-command_script.sh')
- 
+end
 
-step 'SETUP: Configure the code-id script'
+step 'SETUP: Configure the code-id script' do
   on master, 'puppet module install puppetlabs-hocon' 
-  cicsetting=<<-CICS
-    hocon_setting { 'code-id-command-script' :
-      ensure => present,
-      path => '/etc/puppetlabs/puppetserver/conf.d/puppetserver.conf',
-      setting => 'versioned-code.code-id-command',
-      value => '/opt/puppetlabs/server/apps/puppetserver/code-id-command_script.sh',
-      }
-    hocon_setting { 'code-content-command-script' :
-      ensure => present,
-      path => '/etc/puppetlabs/puppetserver/conf.d/puppetserver.conf',
-      setting => 'versioned-code.code-content-command',
-      value => '/opt/puppetlabs/server/apps/puppetserver/code-content-command_script.sh',
-      }
-    CICS
-  create_remote_file(master, '/tmp/config_code_id_command_script.pp', cicsetting)
+  create_remote_file(master, '/tmp/config_code_id_command_script.pp', cicsetting() )
```

Comment:
I don't think the empty parens for a method invocation with an empty parameter list -- or the space between them and the closing paren in the larger parameter list -- are idiomatic. Though it is clearer, and I know that QA has been using styles that are more explicit than idiomatic Ruby usually is. I'd defer to whatever is idiomatic for QA, but figured I should note that.